src/strategies/sections/floor-section.ts

```
@@ -1,28 +1,43 @@
+ */
 export class FloorSectionStrategy {
+  /**
+   * Generate a Lovelace section for the provided floor.
+   *
+   * @param floor - Floor to render.
+   * @returns The section configuration.
+   */
   static async generate(floor: Floor): Promise<LovelaceSectionRawConfig> {
     return {
       type: 'grid',
       cards: await this.generateCards(floor),
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)